index.html

```diff
@@ -1193,30 +1193,39 @@ <h5>Members</h5>
         <dl class="dictionary-members" data-link-for="VideoFrameMetadata" data-dfn-for="VideoFrameMetadata">
           <dt><dfn><code>captureTime</code></dfn> of type <span class="idlMemberType">DOMHighResTimeStamp</span></dt>
           <dd>
-            <p>
-              The capture timestamp of the frame relative to {{Performance}}.{{Performance/timeOrigin}}.
-              MUST be set to the [=capture timestamp=] of the source {{MediaStreamTrack}} video frame if set.
-              It is not [=map/exist|present=] if [=capture timestamp=] is unset.
+            <p>The capture timestamp of the frame relative to {{Performance}}.{{Performance/timeOrigin}}. It corresponds to
```

Comment:
You'd need to add "of the sender" to the first sentence if you also will cover remote tracks.

Reply:
Resolving for now due to other ongoing discussion in https://github.com/w3c/mediacapture-extensions/pull/156#pullrequestreview-2334237029.